includes/Modules/Ads.php

```diff
@@ -19,7 +25,9 @@
  * @access private
  * @ignore
  */
-final class Ads extends Module {
+final class Ads extends Module implements Module_With_Settings, Module_With_Data_Available_State {
+	use Module_With_Settings_Trait;
+	use Module_With_Data_Available_State_Trait;
```

Comment:
Hi @benbowler , was resolving the conflict in my branch, and I saw that `Module_With_Data_Available_State_Trait` has been added. Is there specific reason for it? 

Methods in this trait are used to determine if a module is in gathering data state, but Ads module does not include any report as I am aware. 
cc: @10upsimon @tofumatt

Reply:
@zutigrm you are correct, there is no gathering data state, and probably won't be (at least note for some time until reporting is potentially added in far future phases).